docs/projects/fmcomms2/index.rst

```diff
@@ -77,6 +77,12 @@ Supported carriers
      - `ZedBoard <https://digilent.com/shop/zedboard-zynq-7000-arm-fpga-soc-development-board>`__
      - FMC LPC
 
+.. admonition:: Legend
+   :class: note
+
+   - ``*`` removed; last release that supports this project on this carrier is
+      :git-hdl:`hdl_2023_r2 <hdl_2023_r2:projects/fmcomms2/vc707>`
```

Comment:
```suggestion
   - ``*`` removed; last release that supports this project on this carrier is
     :git-hdl:`hdl_2023_r2 <hdl_2023_r2:projects/fmcomms2/vc707>`
```
One space from the indentation should be removed, because otherwise it still doesn't render it ok